### PR TITLE
Second part of data split for openqa-maintenance bot

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: openqa-maintenance.py [hunk 1]
```diff
@@ -240,127 +242,15 @@ def settings(self, src_prj, dst_prj, packages, req=None):
...
-                'VERSION': '42.3',
-                'FLAVOR': 'Maintenance',
-                'ARCH': 'x86_64',
-                'ISO': 'openSUSE-Leap-42.3-DVD-x86_64.iso',
-            }),
-    ],
-}
+PROJECT_OPENQA_SETTINGS = {}
+
+for i,j in _BASE_POS.items():
```
Comment (coolo) [inline openqa-maintenance.py]: what is BASE_POS and why the _?

Comment (mimi1vx) [inline openqa-maintenance.py]: dictionary filled by incidents.json and '_ ' marks it as internal, but .. You are right , in reality isn't needed

File: openqa-maintenance.py [hunk 2]
```diff
@@ -240,127 +240,16 @@ def settings(self, src_prj, dst_prj, packages, req=None):
...
-                'ARCH': 'x86_64',
-                'ISO': 'openSUSE-Leap-42.3-DVD-x86_64.iso',
-            }),
-    ],
-}
+PROJECT_OPENQA_SETTINGS = {}
+
+with open(opa.join(data_path, "data/incidents.json"), 'r') as f:
+    for i,j in json.load(f).items():
+        if i.split(":")[0] == 'SUSE':
```
Comment (jberry-suse) [inline openqa-maintenance.py]: `i.startswith('SUSE:')`

File: openqa-maintenance.py [hunk 3]
```diff
@@ -240,127 +240,16 @@ def settings(self, src_prj, dst_prj, packages, req=None):
...
+            PROJECT_OPENQA_SETTINGS[i] = [SUSEUpdate(k) for k in j]
+        elif i.split(":")[0] == 'openSUSE':
```
Comment (jberry-suse) [inline openqa-maintenance.py]: `i.startswith('openSUSE:')`

**Comment by coolo** [comment]
Note that you omited the : now - I don't think it matters though - it's unlikely we will have SUSEopen soonish :)

**Pull request merged**